pkg/worker/overworker.go

```diff
@@ -3,6 +3,7 @@ package worker
 import (
 	"context"
 	"fmt"
+	workerConfig "github.com/giongto35/cloud-game/pkg/worker/config"
```

Comment:
Also, Please run `make fmt` (gofmt, goimports) to fix the formatting issues.

Reply:
Done.